lab2.js

```diff
@@ -114,10 +156,14 @@ assert((new Human()).sayHello(new Klingon()) === "nuqneH",
 //*********************************************************
 function max(array) {
   // TODO: return the largest number in the given array
+  return Math.max.apply(Math, array);
```

Comment:
Haha probably not what the author of this assignment intended but points for resourcefulness.